src/components/Home.tsx

```diff
@@ -18,7 +18,13 @@ const Home = () => (
       <Search />
     </Grid>
     <Grid item xs={12}>
-      <HomeButtonsContainer container justify="space-evenly" spacing={8}>
+      <HomeButtonsContainer
+        container
+        direction="row"
+        justify="space-evenly"
+        spacing={8}
+        className="blah"
```

Comment:
wut

Reply:
_You know._ The popular "blah" utility class? Nah, good call that was my bad. [removed](https://github.com/CodeForBoulder/upswyng/pull/86/commits/849c778c7a374de600eb703816e5a1e269d54811).